### ISSUE TITLE
Contributing process

### ISSUE THREAD

**Issue by Leonardofreua** [arxiv-vanity/arxiv-vanity#649]
Is anyone free to submit PR to contribute to the project?

I'm asking this because I didn't see any instructions about contribution.

Thanks.

**Comment by bfirsh** [comment]
Yes, feel free! Sorry for the lack of documentation -- it is due to a lack of my time right now. :)

**Comment by Leonardofreua** [comment]
Oh great. Thanks for the feedback.
